### ISSUE TITLE
ignore superflouous blank lines in readalongs prepare

### ISSUE THREAD

**Issue by joanise** [ReadAlongs/Studio#136]
When a plain text file start with one or more blank lines, blank pages are inserted before the first page of the RA.
When there are 3+ consecutive blank lines mid-text, blank pages are inserted in the middle.
When there are 2+ blank lines at the end of the plain text, blank pages are inserted after the RA.

While we can ask the user to remove such extraneous blank lines, it would be better UX with fewer gotchas to ignore them with these rules:
 - ignore all blank lines before the first non-blank line
 - ignore all blank lines after the last non-blank line
 - consider any sequence of 3+ blank lines the same as a sequence of 2 blank lines, i.e., a single page break.

In short, never create an empty `<div type="page"></div>` element in the output.

**Comment by dhdaines** [comment]
This seems like a good idea, as you say, a page break should be 2 or more blank lines, not 2 blank lines, as it's difficult for a viewer to tell the difference (even 1 vs. 2 blank lines isn't great...)
